org-roam-dailies.el

```diff
@@ -277,13 +278,14 @@ Prefer past dates, unless PREFER-FUTURE is non-nil."
 (defun org-roam-dailies--list-files (&rest extra-files)
   "List all files in `org-roam-dailies-directory'.
 EXTRA-FILES can be used to append extra files to the list."
-  (let ((dir (org-roam-dailies-directory--get-absolute-path)))
+  (let ((dir (org-roam-dailies-directory--get-absolute-path))
+	(regexp (rx "." (eval `(or ,@org-roam-file-extensions)))))
```

Comment:
Oh wow. I completely forgot about existence of `rx`. While being cryptic as regexps, it still allows creating complex expressions in an easier way.